apps/docs/pages/guides/functions/regional-invocation.mdx

```diff
@@ -2,51 +2,54 @@ import Layout from '~/layouts/DefaultGuideLayout'
 
 export const meta = {
   id: 'function-regional-invocation',
-  title: 'Regional Invocation',
+  title: 'Regional Invocations',
   description: 'How to execute an Edge Functions in a particular region.',
+  subtitle: 'How to execute an Edge Functions in a particular region.',
 }
 
-By default, Edge Functions are executed in the region closest to the user making the request. This helps to reduce network latency and provide faster responses to the user.
+Edge Functions are executed in the region closest to the user making the request. This helps to reduce network latency and provide faster responses to the user.
 
-However, if your function does lot of database or storage operations, invoking the function in the same region as your DB may provide better performance.
-
-Some situations where this might be helpful include:
+However, if your Function performs lot of database or storage operations, invoking the Function in the same region as your database may provide better performance. Some situations where this might be helpful include:
```

Comment:
```suggestion
However, if your Function performs lots of database or storage operations, invoking the Function in the same region as your database may provide better performance. Some situations where this might be helpful include:
```